pyproject.toml

```diff
@@ -9,6 +9,7 @@ packages = [{ include = "elasticsearch", from = "custom_components" }]
 
 [tool.poetry.dependencies]
 python = ">=3.12,<3.13"
+elasticsearch8 = "~=8.10.0"
```

Comment:
This also needs to be added to the component's manifest for HASS to load it at runtime:
https://github.com/strawgate/homeassistant-elasticsearch/blob/821319ae7cc969e0667c857253514fa3560715a0/custom_components/elasticsearch/manifest.json#L13